CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.12.0)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
They just released the preview for C++ 23 at the end of last year. C++ 14 is already quite old, why do we need to change it to C++11?